tests/unit_tests/client/proxy/http.cc

```diff
@@ -57,4 +57,31 @@ BOOST_AUTO_TEST_CASE(ok) {
   BOOST_CHECK(tmp.HandleData(tmpData));
 }
 
+BOOST_AUTO_TEST_CASE(JumpServiceI2PAddressHelper) {
+  kovri::client::HTTPMessage tmp;
+  std::string tmp_data = "GET stats.i2p";
+  tmp_data +=
+      // Jump service GET parameter
+      "?i2paddresshelper="
+      // Base64- & URL-encoded I2P address
+ 			"0UVPqAA4xUSfPYPBca24h8fdokhwcJZ4zWvELv-5OsBYTHKtnLzvK7byXtXT~"
```

Comment:
Do you need help setting up clang-format?

Reply:
No, lol. Some derpy literal tab character slid its way in somehow.
Fixing it now.

Reply:
Was the first place I looked after your comment, configuring now. :smile: